lti_consumer/lti_consumer.py

```diff
@@ -421,11 +422,12 @@ class LtiConsumerXBlock(StudioEditableXBlockMixin, XBlock):
         scope=Scope.settings
     )
 
-    # StudioEditableXBlockMixin configuration of fields editable in Studio
-    editable_fields = (
-        'display_name', 'description', 'lti_id', 'launch_url', 'custom_parameters', 'launch_target', 'button_text',
-        'inline_height', 'modal_height', 'modal_width', 'has_score', 'weight', 'hide_launch', 'accept_grades_past_due',
-        'ask_to_send_username', 'ask_to_send_email'
+    # Possible editable fields
+    _editable_fields = (
```

Comment:
Idea was to 'visually' make  it clear that `_editable_fields` should only be used in self implementation and should not be accessed through the instance (or the child instances) and instance should use the corresponding `@property` instead. We're using `@property` as we can easily manipulate actual fields in there.

I can also move this inside the `@property` as local var but that doesn't sound clean to me.